app/components/VaultControls/index.js

```diff
@@ -673,10 +673,10 @@ export default function VaultControls(props) {
       <Wrapper>
         <Box display="flex" flexDirection="column" width={1}>
           {pickleDescriptions.map((description) => (
-            <>
+            <div key={description.main}>
```

Comment:
Ideally we'd use a UID here. In this case the UID is actually description.main. We can't use index as a key as that is not allowed either.